pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -583,6 +529,65 @@ func (o *MustGatherOptions) Run() error {
 	return errors.NewAggregate(errs)
 }
 
+// podProcessor processes Pods created for collecting each must-gather
+func (o *MustGatherOptions) podProcessor(ns string, podChan chan *corev1.Pod, errCh chan error, wg *sync.WaitGroup) {
+	var err error
+	for pod := range podChan {
+		if len(o.RunNamespace) > 0 && !o.Keep {
```

Comment:
Can you put the block under `processNextWorkItem` method? To create something like:
```go
func (o *MustGatherOptions) processNextWorkItem(...) {}

func (o *MustGatherOptions) runWorker(ns string, podChan chan *corev1.Pod, errCh chan error, wg *sync.WaitGroup) {
	for pod := range podChan {
		o.processNextWorkItem(...)
	}
}
```

Reply:
Ack.